static/js/components/SurveyLinkList.jsx

```diff
@@ -18,91 +73,170 @@ SurveyLink.defaultProps = {
   url: null,
 };
 
+SurveyLinkForm.propTypes = {
+  name: PropTypes.string.isRequired,
+  url: PropTypes.string.isRequired,
+  formData: PropTypes.objectOf(PropTypes.any).isRequired,
+  urlEncoded: PropTypes.bool,
+};
+
+SurveyLinkForm.defaultProps = {
+  urlEncoded: false,
+};
+
 const SurveyLinkList = ({ ra, dec, id }) => {
-  const ra_hrs = ra_to_hours(ra);
-  const dec_hrs = dec_to_hours(dec);
-  const thumbnail_timestamp = "TODO";
+  const styles = useStyles();
+  const ra_hrs = ra_to_hours(ra, ":");
+  const dec_hrs = dec_to_dms(dec, ":");
+  // TODO: const thumbnail_timestamp = "TODO";
+  const photometry = useSelector((state) => state.photometry);
+  const objPhotometry = photometry[id];
+
+  let isDetected = false;
+  let magErr = Infinity;
+  let timeStamp = null;
+
+  if (objPhotometry) {
+    objPhotometry.forEach((phot) => {
+      if (phot.mag) {
+        isDetected = true;
+        if (phot.magerr < magErr) {
+          magErr = phot.magerr;
```

Comment:
Where is `magErr` used?

Reply:
L103

Reply:
You mean one line above? That's just the logic that determines its value. But does `magErr` ever get used below?

Reply:
Ah good catch. This is a relic of a previous thing I was doing... but didnt end up keeping thanks

Reply:
Ah wait. No, need to keep this. Magerr is not used below, but we need to use it to update `timeStamp`.

Reply:
(Basically the logic here is an iterative min).

Reply:
Ah, right, I can see that now. :+1: